mixins/_buttons.pcss

```diff
@@ -4,6 +4,7 @@
 
 @define-mixin button-global {
 	border: 0;
+	border-radius: 4px;
```

Comment:
The `button-global` is a button reset, we should apply a `border-radius` to each of the button styles that use it. If later we define another button style without a border radius, this won't work.

Reply:
Hey Paul, thanks for the code review. The whole design system changed and they decided to use curved border buttons. For this design system, the default button is one with curved edges. I guess in the case they later add a squared button, we can add a variation of the class with a modifier for that.

Reply:
Right, I understand. What I'm saying is that we should be adding the `border-radius` to the `button-solid` and `button-border` mixins.

Reply:
I got you wrong then. I thought you wanted to define it as something different and then in common define it for each button we define there.